LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2024
+COPYRIGHT HOLDER: pacta-dashboard-svelte authors
```

Comment:
The list of authors should be documented somewhere (as well as RMI as copyright holder). In R, this is in `DESCRIPTION` (by convention), but I don't know what the common practice is for JS code.

Reply:
Handled in https://github.com/RMI-PACTA/pacta-dashboard-svelte/pull/158